Modules/xSharePoint/DSCResources/MSFT_xSPManagedMetadataServiceApp/MSFT_xSPManagedMetaDataServiceApp.psm1

```diff
@@ -62,6 +64,10 @@ function Set-TargetResource
             
 
             if ($params.ContainsKey("InstallAccount")) { $params.Remove("InstallAccount") | Out-Null }
+            if ($params.ContainsKey("ContentTypeHubUrl")) {
+                $params.Add("HubUri", $params.ContentTypeHubUrl)
```

Comment:
You mean like I did in the MOF description field which goes in to the PowerShell help and the wiki documentation?